src/ts-jest-transformer.ts

```diff
@@ -67,6 +67,10 @@ export class TsJestTransformer implements Transformer {
     return `[object TsJestTransformer<#${this.id}>]`
   }
 
+  /**
+   * Use by e2e, don't mark as internal
+   * @param jestConfig
+   */
```

Comment:
is the `@param` needed when we already have types? The concern is that they might go out of sync.

Reply:
hmm maybe not, webstorm generated it. Will change